DEV.md

```diff
@@ -13,8 +13,9 @@
 
 - Requires that you are listed as a maintainer for this npm module
```

Comment:
Does this mean npm package or GitHub repo?

Reply:
I know these are the instructions to publish to NPM, but it seems like this is a statement about which permissions you need to do that? If so, I don't know that I've ever been granted special privileges via NPM, but I imagine you need to be a maintainer on this project to run this action.

Reply:
The action runs automatically on a new version tag being pushed to this repo.

https://github.com/allen-cell-animated/vole-core/blob/main/.github/workflows/publish.yml

Though I'm wondering if anyone can push a tag and have the NPM version update... Let me test this